src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -149,13 +149,32 @@ object ScalafixPlugin extends AutoPlugin {
       Invisible
     )
 
+  private val scalafixAdaptSbtResolvers: TaskKey[Seq[Repository]] = TaskKey(
+    "scalafixAdaptSbtResolvers",
```

Comment:
scalafix* looks more aligned with previous settings(scalafixResolvers), maybe just `scalafixSbtResolvers`?

Reply:
Right, all internal keys are prefixed with `scalafix` to avoid collisions.... Ideally we would want to reuse existing keys and memoize that in `ThisBuild / resolvers / scalafixResolvers`, but scalafixResolvers is a setting key. I would favor the long but explicit `scalafixSbtResolversAsCoursierRepositories` since I find it weird to have a `scalafixSbtResolvers` returning coursier coursier structures.

Reply:
Reasonable, renamed